src/Swarm/TUI/Controller.hs

```diff
@@ -387,7 +387,7 @@ handleMainEvent ev = do
       -- Paint with the World Editor
       EC.handleCtrlLeftClick mouseLoc
     -- toggle collapse/expand REPL
-    ControlChar 's' -> do
+    ControlChar ',' -> do
```

Comment:
I just realized that, unfortunately, `CTRL + comma` is [not handled properly by the terminal](https://stackoverflow.com/a/7544277/105137).

However `ALT + comma` does work.  So we could change this to 
```suggestion
    MetaChar ',' -> do
```